moveit_ros/moveit_servo/src/pose_tracking.cpp

```diff
@@ -300,8 +300,23 @@ geometry_msgs::msg::TwistStamped::ConstSharedPtr PoseTracking::calculateTwistCom
   return msg;
 }
 
+void PoseTracking::stopMotion()
+{
+  stop_requested_ = true;
+
+  // Send a 0 command to Servo to halt arm motion
+  auto msg = moveit::util::make_shared_from_pool<geometry_msgs::msg::TwistStamped>();
+  {
+    std::lock_guard<std::mutex> lock(target_pose_mtx_);
+    msg->header.frame_id = target_pose_.header.frame_id;
+  }
+  msg->header.stamp = node_->now();
+  twist_stamped_pub_->publish(*msg);
```

Comment:
@AndyZe maybe we should change this to a regular message or a unique_ptr as discussed [here](https://github.com/ros-planning/moveit2/pull/428#discussion_r618461474)

Reply:
Done!